apps/src/templates/instructions/TopInstructionsCSF.jsx

```diff
@@ -46,6 +46,10 @@ const styles = {
     left: 0,
     // right handled by media queries for .editor-column
   },
+  mainCraft: {
```

Comment:
Haha, I had the same thought when I saw the PR. I think it's okay when you see the full code file tho, as it's next to a bunch of other mains.